src/main/java/at/helpch/placeholderapi/expansion/player/util/ItemUtil.java

```diff
@@ -0,0 +1,97 @@
+package at.helpch.placeholderapi.expansion.player.util;
+
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.Damageable;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Optional;
+
+@SuppressWarnings("deprecation")
+public final class ItemUtil {
+
+    /**
+     * Whether an item is null or its type is air (ends with _AIR)
+     *
+     * @param itemStack item to check
+     * @return whether the item is null or air
+     */
+    public static boolean isNullOrAir(@Nullable final ItemStack itemStack) {
+        return itemStack == null || itemStack.getType().name().contains("_AIR");
+    }
+
+    /**
+     * Get the material name of an item
+     *
+     * @param item item
+     * @return {@link Material#name()} or empty string if the item {@link #isNullOrAir(ItemStack) is null or air}
+     */
+    public static @NotNull String getMaterialName(@Nullable final ItemStack item) {
+        return isNullOrAir(item) ? "" : item.getType().name();
+    }
+
+    /**
+     * Get the {@link ItemMeta#getDisplayName() display name} of an item
+     *
+     * @param item item
+     * @return {@link ItemMeta#getDisplayName()} or empty string if the item {@link #isNullOrAir(ItemStack) is null or air}
+     */
+    public static @NotNull String getName(@Nullable final ItemStack item) {
+        if (isNullOrAir(item)) {
+            return "";
+        }
+
+        //noinspection deprecation
+        return item.hasItemMeta() ? item.getItemMeta().getDisplayName() : "";
+    }
+
+    public static int getData(@Nullable final ItemStack item) {
```

Comment:
Should probably start calling this getDamage since that's the only thing it does on newer versions right?